public_suffix_list.dat

```diff
@@ -15755,4 +15755,8 @@ basicserver.io
 virtualserver.io
 enterprisecloud.nu
 
+// frii.site : https://www.frii.site
+// Submitted by Onni Nevala <nevalaonni@frii.site>
+*.frii.site
```

Comment:
Since your subdomain service allows people to register subdomains of `frii.site`, I assume you would want to add `frii.site` instead of `*.frii.site` to the PSL. The wildcard character matches any valid sequence of characters in a hostname part, so you would likely need to remove the `*.` part.

```suggestion
frii.site
```